installer/bundle/bundle_skel.sh

```diff
@@ -801,6 +802,16 @@ case "$installMode" in
                     [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
                 fi
             done
+			echo "----- Installing MySQL ruby gem -----"
+			set +e
+			$OMSAGENT_RUBY_DIR/gem install mysql2 2>/dev/null
```

Comment:
And finally, do you want to install blindly? Or do you want to check prereqs and only try and install if you know it'll actually work?